test/Libraries/SystemTestServices/SystemTestBase.cs

```diff
@@ -305,15 +306,15 @@ public void AssertPreviewCount(string guid, int count)
             Assert.AreEqual(count, data.GetElements().ToList().Count);
         }
 
-        public NodeModel GetNode<T>(string guid) where T : NodeModel
+        protected NodeModel GetNode(string guid)
```

Comment:
nunit wants to resolve types from public methods at the discovery stage.
Making this protected allows discovery to run successfully without having all dynamo assemblies included in the test outout directory

Reply:
so this will yield a runtime failure now if the assembly is not resolved, but at least we'll discover the test?

Reply:
Yes, it will trigger a runtime exception now (instead of compile time) if the T type is not derived from NodeModel.
I marked the method as obsolete so we will be able to replace it with something else that will not cause the same issue